src/integration-tests/workload/deploy_workload_test.go

```diff
@@ -39,9 +39,28 @@ var _ = Describe("Deploy workload", func() {
 
 	})
 
+        It("allows access to pod logs", func() {
+
+                deployNginx := runner.RunKubectlCommand("create", "-f", nginxSpec)
+                Eventually(deployNginx, "60s").Should(gexec.Exit(0))
+                rolloutWatch := runner.RunKubectlCommand("rollout", "status", "deployment/nginx", "-w")
+                Eventually(rolloutWatch, "120s").Should(gexec.Exit(0))
+
+                getPodName := runner.RunKubectlCommand("get", "pods", "-o", "jsonpath='{.items[0].metadata.name}'")
+                Eventually(getPodName, "5s").Should(gexec.Exit())
+                podName := string(getPodName.Out.Contents())
+
+                getLogs := runner.RunKubectlCommand("logs", podName)
+                Eventually(getLogs, "5s").Should(gexec.Exit())
+                logContent := string(getLogs.Out.Contents())
+                // nginx pods do not log much, unless there is an error we should see an empty string as a result
+                Expect(logContent).To(Equal(""))
+
+        })
+
 	AfterEach(func() {
 		session := runner.RunKubectlCommand("delete", "-f", nginxSpec)
-		session.Wait("30s")
+		session.Wait("60s")
```

Comment:
This is happening as are using a hard coded NodePort rather than dynamically resolving it. Create a chore to change this - https://www.pivotaltracker.com/story/show/149952987